seisbench/models/pickblue.py

```diff
@@ -0,0 +1,15 @@
+from .eqtransformer import EQTransformer
+from .phasenet import PhaseNet
+
+
+class PickBlue:
```

Comment:
@borthom I've not used `__new__` before, so I'm unsure about the code. Is there any (dis)advantage to something like the following?

```python
def PickBlue(base="eqtransformer"):
   if base == "eqtransformer":
      return EQTransformer.from_pretrained("obs")
...

```

Reply:
Done.

Reply:
I don't think so! I just wanted to stick to the pattern of picker classes. But it really makes no big difference, so why not use a function instead? It's also more readable.

Reply:
Adjusted this now